version.txt

```diff
@@ -1 +1 @@
-0.6.24
```

Comment:
It looks we have the convention to cut release in a separate PR, not sure if it can work if we do it together with other changes

Reply:
Thank for the heads up - I removed the change from the PR.

Once this PR is merged, I'll send a separate PR to bump the version again.

Reply:
PR to only bump the version: https://github.com/kubernetes-sigs/aws-iam-authenticator/pull/748